lib/jellyfish_web/api_spec/component/recording.ex

```diff
@@ -17,12 +17,14 @@ defmodule JellyfishWeb.ApiSpec.Component.Recording do
       description: "Properties specific to the Recording component",
       type: :object,
       properties: %{
-        pathPrefix: %Schema{
+        subscribeMode: %Schema{
           type: :string,
-          description: "Path prefix under which all recording are stored"
+          description:
+            "Whether the Recording component should subscribe to tracks automatically or manually",
+          enum: ["auto", "manual"]
         }
       },
-      required: [:pathPrefix]
```

Comment:
Why are we removing pathPrefix here?

Reply:
This was a change requested by the Cloud team.

Reply:
OK, why are we keeping `path_prefix` in `properties` of the Recording endpoint [when creating one](https://github.com/jellyfish-dev/jellyfish/blob/83c7a6c84a6051a31e8df93f6f9db2caa88fa451/lib/jellyfish/component/recording.ex#L50) then? All properties get serialised to JSON, so ATM we don't conform to the schema